go.mod

```diff
@@ -13,5 +13,6 @@ require (
 	github.com/multiformats/go-multiaddr v0.0.2
 	github.com/multiformats/go-multihash v0.0.1
 	github.com/spacemonkeygo/openssl v0.0.0-20181017203307-c2dcc5cca94a
-	golang.org/x/crypto v0.0.0-20190225124518-7f87c0fbb88b
+	go.opencensus.io v0.21.0
+	golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2
```

Comment:
Let's not slip in unrelated dependency upgrades.

Reply:
This is a result of running `go mod tidy`, at request of @bigs. Cole, I'm undoing that

Reply:
@raulk yeah, not super sure how to handle things like this. in my experience doing any work with the `go` tool is liable to add things to `go.mod` as is using the language server, etc. i've had him `go mod tidy` before commits, but these things seem to bleed in. i think they're okay, as the mental overhead of pruning them would seem to be a lot.